changelogs/unreleased/4200-lrewega-small.md

```diff
@@ -0,0 +1 @@
+Removes spec.ttlSecondsAfterFinished from certgen job in versioned releases, as immediately deleting it upon competition will not be useful for most consumers.
```

Comment:
Ah, can't believe I missed this. Sorry.
```suggestion
Removes spec.ttlSecondsAfterFinished from certgen job in versioned releases, as immediately deleting it upon completion will not be useful for most consumers.
```

Reply:
Ah jeez, _I_ can't believe I _did_ this. Just one of those days...